io/include/pcl/io/hdl_grabber.h

```diff
@@ -152,12 +152,17 @@ namespace pcl
       filterPackets (const boost::asio::ip::address& ipAddress,
                      const unsigned short port = 443);
 
-      /** \brief Allows one to customize the colors used for each of the lasers.
+      /** \brief Allows one to customize the colors used for each number of the lasers.
```

Comment:
* Switch to `Allows one to customize the colors used by each laser.`.
* Add the documentation for the input parameter.